### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by eliboa** [eliboa/TegraRcmGUI#24]
_TODO : Translate to english._

Dans la navigation au clavier, les onglets et le reste des contrôles sont inaccessible de l'un en vers l'autre, par exemple si je navigue dans les onglets la touche "tab" ne me permet pas de passer aux autres contrôles de la fenêtre et quand je suis sur le reste c'est les onglets qui ne sont pas accessibles avec la touche "tab", déjà un premier palliatif pourrait être d'implémenter un raccourci clavier "ctrl+tab" pour passer d'un onglet à un autre mais bon pouvoir quitter la zone des onglets ou y aller avec la touche "tab" serait tout de même une bonne chose à régler.

**Comment by Abu-ZaydAbdullah** [comment]
In the keyboard navigation, the tabs and the rest of the controls are inaccessible from one to the other, for example if I navigate in the tabs the "tab" key does not allow me to go to the other controls of the window and when I am on the rest it is the tabs which are not accessible with the "tab" key, a first patch could be to implement a keyboard shortcut "ctrl + tab" to move from one tab to another, but being able to leave the tab area or go there with the "tab" key would still be a good thing to adjust.
